src/commands/schema/status.mjs

```diff
@@ -15,33 +16,38 @@ async function doStatus(argv) {
   const logger = container.resolve("logger");
   const makeFaunaRequest = container.resolve("makeFaunaRequest");
 
-  let params = new URLSearchParams({ diff: "summary" });
   const secret = await getSecret();
+  const absoluteDirPath = path.resolve(argv.dir);
   const gatherFSL = container.resolve("gatherFSL");
   const fsl = reformatFSL(await gatherFSL(argv.dir));
 
+  const hasLocalSchema = fsl.entries().next().done === false;
+
+  const statusParams = new URLSearchParams({ diff: "summary" });
```

Comment:
There is no `diff` param documented for `/status` or `/diff` endpoints. Is the `format: "summary"` the param you want?

Reply:
Can confirm — `format` is the new alias for `diff`.